### PR TITLE
add faqs to sidebar

### PR DESCRIPTION
**Pull request by suhailkakar**

### CODE REVIEW

File: references/knowledge-base/playback.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    <Player
+      title="My Livestream"
+      playbackId="wwl88k4g8f5eese9"
+      lowLatency="force"
+    />
+  );
+}
+```
+
+2. If you are using the iframe embeded player, you can add `lowLatency=true` to the query string.
```
Comment (Shih-Yu) [inline references/knowledge-base/playback.mdx:45]: This section does not appear to be part of the card, maybe a syntax issue?

**Review by Shih-Yu — CHANGES_REQUESTED** [review]
Let a review

File: references/knowledge-base/playback.mdx (hunk 1 above)
Comment (suhailkakar) [reply references/knowledge-base/playback.mdx:45]: Actually it is a part of card :)

image

Comment (Shih-Yu) [reply references/knowledge-base/playback.mdx:45]: The code just doesn't show the highlighted colors.

**Review by Shih-Yu — APPROVED** [review]
LGTM

File: references/knowledge-base/playback.mdx (hunk 1 above)
Comment (suhailkakar) [reply references/knowledge-base/playback.mdx:45]: Fixed #503

**Pull request merged**